chia/full_node/mempool_manager.py

```diff
@@ -246,9 +242,16 @@ async def pre_validate_spendbundle(
         start_time = time.time()
         if new_spend_bytes is None:
             new_spend_bytes = bytes(new_spend)
-        err, cached_result_bytes, new_cache_entries = await asyncio.get_running_loop().run_in_executor(
-            self.pool,
-            validate_clvm_and_signature,
+
+        # err, cached_result_bytes, new_cache_entries = await asyncio.get_running_loop().run_in_executor(
```

Comment:
Why are we not running this in the executor? This might cause issues since it's a lot of work for the main process.

Reply:
There is CLVM raise issue, switched for debugging purposes. Will be switched back before removing draft tag.